MatrixSDK/JSONModels/MXJSONModels.m

```diff
@@ -628,6 +628,11 @@ + (id)modelFromJSON:(NSDictionary *)JSONDictionary
     return pushRule;
 }
 
+- (NSString *)description
+{
+    return [NSString stringWithFormat:@"<MXPushRule: %p> ruleId: %@ - isDefault: %@ - enabled: %@ - actions: %@", self, _ruleId, @(_isDefault), @(_enabled), _actions];
```

Comment:
You can use `%d` for Bool value to avoid create an intermediate NSNumber

Reply:
We use %@ for everything to avoid bad formatting.

There is also the hope for the particular case of BOOL, that, one day, NSNumber will be able to return YES or NO as description.